### PR TITLE
Add showInAppNotification position param support for Android

### PR DESCRIPTION
**Pull request by Royce**
Allows position parameter to make the notification appear on bottom of Screen. Defaults to `top`.

```javascript
Navigator.showInAppNotification({
    screen: "example.InAppNotification",
    position: "bottom",
});
```

### CODE REVIEW

File: android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java [hunk 1]
```diff
@@ -9,17 +9,21 @@
 
 public class PeekingAnimator {
 
+    public enum Position {
```
Comment (Royce) [inline android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java]: I wasn't entirely certain where this Position Enum should live.

Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java]: Think it should be in `SlidingOverlayParams`

File: android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java [hunk 2]
```diff
@@ -11,6 +12,14 @@ public SlidingOverlayParams parse(Bundle bundle) {
         final SlidingOverlayParams result = new SlidingOverlayParams();
         result.screenInstanceId = bundle.getString("screen");
         result.navigationParams = new NavigationParams(bundle.getBundle("navigationParams"));
+        switch(bundle.getString("position", "top")) {
```
Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java]: The `position` parsing logic can be extracted to a method

File: android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java (hunk 1 above)
Comment (Royce) [inline android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java]: Damn. It started off there, but it seemed weird to use it in PeekingAnimator.

File: android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java (hunk 2 above)
Comment (Royce) [inline android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java]: Will do.

File: android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java (hunk 1 above)
Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/animation/PeekingAnimator.java]: No problem, your call 👍

File: android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java (hunk 2 above)
Comment (guyca) [inline android/app/src/main/java/com/reactnativenavigation/params/parsers/SlidingOverlayParamsParser.java]: great!

**Pull request merged**